jre/guide/java/java_jre_enable_jws_locked/oval/jre.xml

```diff
@@ -0,0 +1,24 @@
+<def-group>
+  <definition class="compliance" id="java_jre_enable_jws_locked" version="1">
+    <metadata>
+      <title>Disable User Access to Java Web Start Application Setting</title>
+      <affected family="unix">
+        <product>Java Runtime Environment</product>
+      </affected>
+      <description>The setting that ensures Java allows Java 
+      Web Start applications to run must be locked.</description>
```

Comment:
@carlosmmatos can you expand the description like what is done in the others to tell people how this is done?

Reply:
@redhatrises When you say others, can you show me what you mean? This was templated off the other *_locked oval files. The description is short and sweet, no additional verbiage seems to be added to other controls in this section.